text/0000-discord.md

```diff
@@ -0,0 +1,178 @@
+
+**Category/Channel List:**
+- (No Category)
+  - welcome (community guidelines are posted here)
+  - setup-profile (how to setup your profile)
+- Admin
+  - community-feedback (questions, comments, concerns, requests)
+  - security
+  - steering-committee 🔒 (locked to role “steering-committee”)
+  - news & announcements
+  - ember-jobs
+  - bots
+- Core Teams
+  - ember-js 🔒 (locked to role “core-js”)
+  - ember-data 🔒 (locked to role “core-data”)
+  - ember-cli 🔒 (locked to role “core-cli”)
+  - ember-learning 🔒 (locked to role “core-learning”)
+- Working on Ember
+  - ember-cli
+  - ember-data
+  - ember-engines
+  - ember-js
+  - glimmer-vm
+  - triage
+  - st-* (as needed)
+- Using Ember
+  - general-help
+  - learning-ember
+  - a11y
+  - backend
+  - internationalization
+  - jsonapi
+  - mobile
+  - ember-js
+  - ember-data
+  - ember-cli
+  - ember-engines
+  - fastboot
+  - ember-twiddle
+  - e-*
+- Supporting Ember
+  - documentation
+  - website
+  - marketing-and-advocacy
+  - infrastructure
+- Event-Chat
+  - EmberConf
+  - EmberCamps
+  - EmberFest
+  - Talks
+  - Other Conferences
+  - Meetup organizers
+- Social
+  - Water-cooler (random)
+  - Local-*
+  - Media (livestreams, videos, podcasts)
+  - Pets
+  - Women in Ember 🔒
+  
+**Integrations**
+Discord's integration game is strong. Discord has a [very detailed API](https://discordapp.com/developers/docs/intro) and many integrations already exist, and with no limitation (compared to free Slack instances, that have limited numbers of integrations).
+
+## How do we teach this?
+
+In addition to having a setup channel available upon login (with illustrated instructions), here are some links where community members can read more: 
+
+- [Discord Loves Open Source](https://discordapp.com/open-source)
+- [Discord Community Guidelines](https://discordapp.com/guidelines)
+- [How to use Discord](http://www.businessinsider.com/how-to-use-discord-the-messaging-app-for-gamers-2018-5)
+
+## Drawbacks
+
+### Supporting Learning vs Supporting Development
+
+There is some concern that there is already some confusion on Slack about where to get help learning/using Ember, and where to coordinate working on Ember. We need to have a clear delineation so that the folks who are spending their volunteer time to ship Ember features can continue to concentrate and do that.
+
+### Losing Community Members 
+
+There is some concern that we may lose some community members due to this move. This could happen for a variety of reasons- the nature of OSS work means that some are not always active on the chat community, or the user doesn't want a different chat app, etc. We believe that the former is probably more likely than the latter, since many of us are on at least 2-3 chat apps already. 
+
+## Alternatives
```

Comment:
An open source alternative such as https://www.mattermost.org/ should at least be considered. I did this analysis ~3 years ago for a community and it wasn't there yet, but it might be today.

Reply:
Mattermost is an interesting project, but I don't think it's feasible for a volunteer-run open source project to get into the business of self-hosting critical infrastructure. Not only does it take away time and resources from other important work, I would be extremely uncomfortable running a production service without a full-time ops team taking care of security patches, performance optimization, backups, etc. Outsourcing this to a third party is the only option that makes sense for a project of our size, IMO.

Reply:
That's fine, I just think it should be in this section at least.

Reply:
As an open source alternative, there is also https://rocket.chat/. However, that means you need to maintain your infra to support the chat.

Reply:
I think anything that requires infrastructure management is out of the question. 

Community chat should be _simple_